### DISCUSSION TITLE
中身が「移転しました」と外部リンクのみになっている記事について

### DISCUSSION THREAD

**Discussion by orino-akira** [increments/qiita-discussions#271] [要望・アイデア・不具合]
qiitaのSEOに便乗して検索上位に残って、信用できるかもわからないドメインに誘導する不愉快な記事でしかない認識なんですが、運営としてはどうお考えですか？

個人的には全て消えてもらいです。

**Comment by tanaka-asami415** [comment]
@esui-minato 
お問い合わせありがとうございます🙏

記事の内容と直接的に関係ないURLに誘導したり、
記事上で完結させることも出来る情報を敢えて載せずに、外部のサイトに誘導することは
利用規約違反となりますので順次対応を行っております。

対応しきれていない部分もあり、ご不便をおかけして申し訳ありません。今後はより対応を強化していきます。
利用規約やコミュニティガイドライン違反の報告につきましては、通報機能をご利用いただけますと幸いです🙇

**Reply by orino-akira** [reply]
見解について承知しました。

検索しているときの目的はパトロールではなくて情報収集なので積極的に通報を行うことはないと思いますが、、、

**Comment by eduidl** [comment]
私もああいうSEOへの便乗は良くないと思う反面、リンク切れ防止という面もあるので、放置・消し去る以外の別の可能性があると嬉しいなと思います。

**Comment by tmura00** [comment]


**Reply by tmura00** [reply]
> 利用規約やコミュニティガイドライン違反の報告につきましては、通報機能をご利用いただけますと幸いです🙇

通報も何度も行っていますが、通報は本当に届いていますか？
